src/MSBuild/XMake.cs

```diff
@@ -766,6 +769,40 @@ string commandLine
 
             return exitType;
         }
+
+        /// <summary>
+        /// Generates a markdown file on disk with the result of profiling the evaluation
+        /// </summary>
+        private static void GenerateProfilerReport(ProfilerLogger profilerLogger)
```

Comment:
Should this logic go in the ProfileLogger? On shutdown or something?

Reply:
I intendedly kept the file saving part outside of the profiler logger because I was seeing the logger more as a collector of data. But I can move the saving logic inside.